src/aiortc/contrib/media.py

```diff
@@ -323,11 +321,13 @@ class MediaPlayer:
     :param options: Additional options to pass to FFmpeg.
     """
 
-    def __init__(self, file, fps=None, save_dir=None, format=None, options={}):
+    def __init__(self, file, enable_prediction=False, reference_update_freq=30, fps=None, save_dir=None, format=None, options={}):
```

Comment:
this line maybe too long too - can't tell from this view - please check

Reply:
shortened, thanks